src/main.c

```diff
@@ -6,7 +6,8 @@
 
 #include <stdlib.h>
 #include <stdint.h>
-#include <emmintrin.h>
+#define SIMDE_ENABLE_NATIVE_ALIASES
```

Comment:
Can you `#undef` this before the header ends? I don't think we want to leak the define to headers that include this one.

Reply:
Done, good point